src/components/Header/HeaderImage/HeaderImage.scss

```diff
@@ -30,7 +30,8 @@
 
       background-image: url("../../../assets/headerImage.JPG");
       min-height: 700px;
-      background-attachment: fixed;
+      background-size: 100vh auto !important;
```

Comment:
let's not use !important in css 
and using viewport height to set max height gets tricky because it can cause some weird overflow issues

Reply:
can you show me the overflow issues? it looks good on my end since its not stretching on scroll anymore